.github/workflows/blank.yml

```diff
@@ -95,6 +95,17 @@ jobs:
           script: |
             mkdir -p /var/www/csr/stage/
             echo '${{ secrets.DEPLOY_CONFIG }}' > /var/www/csr/stage/config.json
+            sudo rm -rf /etc/systemd/system/stage.csr.service.d
```

Comment:
I think `EnvironmentFile` directive might be simplier to work with compared to folder management.
https://www.flatcar.org/docs/latest/setup/systemd/environment-variables/
Could you try this approach and see how it works?
Please create environment file in the same directory with `stage.csr.service` - do not create unnecessary folders as possible.

Reply:
Thanks for your advice! Updated this flow. Here are successful run: https://github.com/CSR-LC/csr-be/actions/runs/11105969158